src/lib/utils/gh.ts

```diff
@@ -17,5 +17,8 @@ export const getLastModifiedDate = async (filePath: string) => {
   return fetch(url, { headers })
     .then((res) => res.json())
     .then((commits) => commits[0].commit.committer.date)
-    .catch(console.error)
+    .catch((err) => {
+      console.error(err)
+      return ""
```

Comment:
why are we returning an empty string here?

Reply:
Oh good catch, forgot to remove this.

I was getting an error on build time randomly. Sometimes, the fetch fails and that breaks the build which is a correct behavior but maybe we could do something different in dev mode as it is not that critical data for development.